hash/gbcolor.xml

```diff
@@ -28397,4 +28397,59 @@ license:CC0-1.0
 			</dataarea>
 		</part>
 	</software>
+
+
+<!-- Aftermarket games -->
+
+	<software name="grimacesb">
+		<description>Grimace's Birthday (rev 2)</description>
+		<year>2023</year>
+		<publisher>Krool Toys</publisher>
+		<notes><![CDATA[
+Officially licensed free game, released as part of McDonald's "Grimace's Birthday" 2023 campaign.
+ROM name was taken from the original file used by the website's emulator.
+]]></notes>
```

Comment:
We don’t need these “comments that aren’t comments” in the notes field.  It’s just clutter.

Reply:
Ok, they're gone.

Reply:
Though still, it might be a good thing to mention somewhere that it's an official game (and not as "publishers" since they're not the ones distributing it).